### ISSUE TITLE
Active speaker display switching too slow for short sentences

### ISSUE THREAD

**Issue by jbalint-de** [wireapp/wire-webapp#10304]
Wire version: 3.21.3959
Wire for web version: 2021.01.18.16.46
Operating system: macOS BigSur 11.1

What steps will reproduce the problem?
1. Join a video group call
2. Select to show only the active speakers video
3. Speak short sentences

What is the expected result?
Video of the speaker speaking only a very short time should not be shown or instantly be shown (what may be difficult). 

What is the actual result?
When you have a conversation without long monologues you often see a speaker way too late or have the grey switching "there's no active" screen way too often. It's really distracting and lowers the productivity of a meeting when your meeting is too fast for Wire to handle the switching between active speakers.

**Comment by jspittka** [comment]
Thanks for this report! We are aware that this first iteration isn't perfect. Please stay tuned for more improvements in the next releases. In the meantime please use the "All" view to maintain your productivity in meetings. That one should perform well?!

**Comment by jbalint-de** [comment]
Yeah, the "All" view performs well except for the little bug I issued in #10305. 😉

**Comment by jspittka** [comment]
This should be improved in the latest version. Please let us know how it works for you.

**Comment by jbalint-de** [comment]
Still doesn't work very well for me. It kind of improved (at least feels like it since I can't accurately measure it) but still the switching from non-active to active is way too slow for my (and some of my contacts) taste. In comparison to - for example the Signal desktop client - Wire still feels a bit less responsive. For some participants the speech recognition doesn't work at all and the activity indicator keeps beeing on/off even if they (don't or didn't) speak.

**Comment by jspittka** [comment]
Thank you for the feedback! We currently work on some performance improvements that should improve the activity indication. Switching out active videos in the active speaker view potentially takes some time though since we have to reinitialize the video streams due to the end-to-end encryption. Here is also some room for improvements in the future. We will keep you posted.
